CCPA/Data Deletion Request Handling.md

```diff
@@ -0,0 +1,328 @@
+
+```
+identifiers {
+  ...
+  "platform": "ios-app-store",
+  "app_identifier": "01234567891446075923",
+  "user_identifier": "AEA12347583AACD-A123667-A418AABC-AB123806-1242AEAACB12AB1234548606"
+}
+...
+{
+  "platform": "android-play-store",
+  "app_identifier": "com.acmeinc.acmeapp",
+  "user_identifier": "aaaacdda123802ae-11fb191c-1247abad-12340794ad123394ac123912"
+...
+}
+```
+
+```
+Note: For purposes of this explanation, a "bundle id" is used for (iOS) and a "package" for (Android).
+```
+
+### Common Platform / Stores identifiers
+
+These identifiers can be used for the "platform" field where applicable. Otherwise, the platform field can include an arbitrary value.
+
+<table>
+  <tr>
+   <td>Android Google Play Store
+   </td>
+   <td>android-play-store
+   </td>
+  </tr>
+  <tr>
+   <td>Android Amazon Store
+   </td>
+   <td>android-amazon-store
+   </td>
+  </tr>
+  <tr>
+   <td>iOS App Store
+   </td>
+   <td>ios-app-store
+   </td>
+  </tr>
+  <tr>
+   <td>Samsung App Store
+   </td>
+   <td>samsung-app-store
+   </td>
+  </tr>
+  <tr>
+   <td>Huawei app store
+   </td>
+   <td>huawei-app-store
+   </td>
+  </tr>
+  <tr>
+   <td>Sony apps
+   </td>
+   <td>sony-app-store
+   </td>
+  </tr>
+  <tr>
+   <td>LG smartworld
+   </td>
+   <td>lg-app-store
+   </td>
+  </tr>
+</table>
+
+
+## Example Implementation
+
+Publisher site setup:
+
+```
+<html>
+  <head>
+    <script>
+      //API provided by the publisher or CMP:
+      function uspapi_addFrame()
+      {
+       if(!window.frames['__uspapiLocator'])
+       {
+        if(document.body)
+        {
+         var i = document.createElement('iframe');
+         i.style.cssText = 'display:none';
+         i.name = '__uspapiLocator';
+         document.body.appendChild(i);
+        }
+        else
+        {
+         window.setTimeout('uspapi_addFrame()', 10);
+        }
+       }
+      }
+      
+      function uspapi_stub()
+      {
+       var b = arguments;
+       __uspapi.a = __uspapi.a || [];
+       if(!b.length)
+       {
+        return __uspapi.a;
+       }
+       else
+       {
+        __uspapi.a.push([].slice.apply(b));
+       }
+      }
+      
+      function uspapi_msghandler(event)
+      {
+       var msgIsString = typeof event.data === 'string';
+       try
+       {
+        var json = msgIsString ? JSON.parse(event.data) : event.data;
+       }
+       catch(e)
+       {
+        var json = null;
+       }
+       if(typeof (json) === 'object' && json !== null && '__uspapiCall' in json)
+       {
+        var i = json.__uspapiCall;
+        window.__uspapi(i.command, i.version, function (retValue, success) 
```

Comment:
Does this need the extra optional parameter ('identifiers') used in non-web contexts as a placeholder?

Reply:
yes, probably

Reply:
can you show exactly where that would go?

Reply:
No it doesn't.  Param is the last value and it's optional, so no need to pass it unless it is defined.

Reply:
It's already there I think. I may have missed it in my original comment or it may have been added (not honestly sure which). It's on line 229 below.